File: projects/web/java/spring/{{cookiecutter.project_slug}}/src/main/java/{{cookiecutter.java_package_dir}}/api/HealthController.java
package {{ cookiecutter.java_package }}.api;

import {{ cookiecutter.java_package }}.Version;
import {{ cookiecutter.java_package }}.dto.Health;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

/** Health. Every API exposes it at the root. */
@RestController
public class HealthController {
    @GetMapping("/health")
    public Health health() {
        return new Health("ok", Version.VERSION);
    }
}

File: projects/web/kotlin/spring/{{cookiecutter.project_slug}}/src/main/kotlin/{{cookiecutter.java_package_dir}}/api/HealthController.kt
package {{ cookiecutter.java_package }}.api

import {{ cookiecutter.java_package }}.Version
import {{ cookiecutter.java_package }}.dto.Health
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

/** Health. Every API exposes it at the root. */
@RestController
class HealthController {
    @GetMapping("/health")
    fun health() = Health("ok", Version.VERSION)
}
